src/components/active-object/index.jsx

```diff
@@ -120,7 +120,8 @@ const ActiveObject = ({ object, handleSavedObjectChange, savedObjects }) => (
 					<a
 						target="_blank"
 						className="active-object__link"
-						href={object.primaryImage}>
+						href={object.primaryImage}
+						rel="noreferrer">
```

Comment:
curious why you added noreferrer here?

Reply:
https://web.dev/external-anchors-use-rel-noopener/

Just something the linter made me aware of.